scan.py

```diff
@@ -156,6 +177,9 @@ def lambda_handler(event, context):
     set_av_tags(s3_object, scan_result)
     sns_scan_results(s3_object, scan_result)
     metrics.send(env=ENV, bucket=s3_object.bucket_name, key=s3_object.key, status=scan_result)
+    # Move Items with clean results to destination bucket
+    if scan_result == AV_STATUS_CLEAN:
```

Comment:
Could we add a test to only try the move if `DESTINATION_BUCKET` is set? Current logic would output the _Destination Bucket does not exist!_ message for users that don't have this option enabled.

Reply:
sure thing.

Reply:
Added in test to only call move function if the destination bucket parameter is set and the file is clean.